examples/ssl-2way-project/build.gradle

```diff
@@ -77,10 +78,11 @@ if (project.hasProperty("mlKeystore")) {
       KeyManager[] key = keyManagerFactory.getKeyManagers()
 
       // Initialize the SSL context with key and trust managers.
-      SSLContext sslContext = SSLContext.getInstance("SSLv3")
+      SSLContext sslContext = SSLContext.getInstance("TLSv1.2")
       sslContext.init(key, trust, null)
 
       restSslContext = sslContext
+			restTrustManager = trust[0]
```

Comment:
Is it possible for trust to be an empty array?

Reply:
I believe that would require using a JVM with an empty cacerts file with no trust managers. That seems very unlikely, and given that this is only example code, that's something a user would get a hard error on - which is probably better than failing by not passing in any trust manager at all.